data_model/src/query/mod.rs

```diff
@@ -1203,12 +1203,28 @@ pub mod http {
             pub fetch_size: FetchSize,
         }
 
+        /// Signature of query
+        #[derive(
+            Debug,
+            Clone,
+            PartialEq,
+            Eq,
+            PartialOrd,
+            Ord,
+            Decode,
+            Encode,
+            Deserialize,
+            Serialize,
+            IntoSchema,
+        )]
+        pub struct QuerySignature(pub PublicKey, pub SignatureOf<ClientQueryPayload>);
```

Comment:
After Sato's work pub key can be extracted from `AccoundId`.

Reply:
done, although why do we even sign queries?

Reply:
We have query permissions if we want to make sure who makes query we have to sign it.